test/commands/add.test.js

```diff
@@ -15,29 +15,79 @@ test('right usage', t => {
 
   cli.on('close', (code) => {
     const ks = h.readKeepyStore()
-    t.equals(code, 0)
-    t.equals(ks.data.length, 1)
-    t.deepEquals(ks.data[0].tags, [])
+    t.equal(code, 0)
+    t.equal(ks.data.length, 1)
+    t.same(ks.data[0].tags, [])
   })
 })
 
-test('import file', t => {
+test('import file env', t => {
   t.plan(5)
   const importFile = '.env'
   writeFileSync(importFile, 'HELLO=WORLD\nCIAO=MONDO\nHOLA=MUNDO\n')
   h.createTestKeepyStore()
   const cli = spawn(node, ['cli', 'add', '-f', importFile, '-t', 'from-file', '-w', 'ciao'])
   cli.on('close', (code) => {
     const ks = h.readKeepyStore()
-    t.equals(code, 0)
-    t.equals(ks.data.length, 3)
-    t.equals(ks.data[0].tags.length, 1)
-    t.equals(ks.data[1].tags.length, 1)
-    t.equals(ks.data[2].tags.length, 1)
+    t.equal(code, 0)
+    t.equal(ks.data.length, 3)
+    t.equal(ks.data[0].tags.length, 1)
+    t.equal(ks.data[1].tags.length, 1)
+    t.equal(ks.data[2].tags.length, 1)
   })
 })
 
-test('update imported file', t => {
+test('import file json plain', t => {
+  t.plan(5)
+  const importFile = '__test__.json'
+  writeFileSync(importFile, '{\n"HELLO":"WORLD",\n"CIAO":"MONDO",\n"HOLA":"MUNDO"\n}\n')
+  h.createTestKeepyStore()
+  const cli = spawn(node, ['cli', 'add', '-f', importFile, '-t', 'from-file', '-w', 'ciao'])
+  cli.on('close', (code) => {
+    const ks = h.readKeepyStore()
+    t.equal(code, 0)
+    t.equal(ks.data.length, 3)
+    t.equal(ks.data[0].tags.length, 1)
```

Comment:
Could you add a check on the data value?

I think there could be some extra quotes when reading the store

```
JSON.stringify("hello")
// '"hello"'
```

Reply:
Good point, I'll fix it